orttraining/orttraining/python/training/torchdynamo/ort_backend.py

```diff
@@ -394,19 +406,19 @@ def __init__(self):
         self.sessions: Dict[torch.fx.GraphModule, onnxruntime.InferenceSession] = {}
         # self.input_names[mod] contains all input names in the ONNX model exported from mod.
         # self.input_names[mod][i] is the name of the i-th positional input of the graph in mod.
-        self.input_names: Dict[torch.fx.GraphModule, Tuple[str]] = {}
+        self.input_names: Dict[torch.fx.GraphModule, Tuple[str, ...]] = {}
         # Similar to self.input_names, but for outputs of the graph.
-        self.output_names: Dict[torch.fx.GraphModule, Tuple[str]] = {}
+        self.output_names: Dict[torch.fx.GraphModule, Tuple[str, ...]] = {}
         # self.input_devices[mod] contains devices of inputs fed to mod.forward (excluding self).
```

Comment:
We data classes will create the init for us: I would use `default_factory=set` https://docs.python.org/3/library/dataclasses.html#dataclasses.field to accomplish this.

Reply:
Nice to know! But I prefer to keep them so other people (who are not familiar with dataclass) see those are initialized. If this class is too big, using dataclass for implicit initialization will be a great idea.